.github/workflows/emscripten.yml

```
@@ -21,13 +21,13 @@ jobs:
     runs-on: ubuntu-22.04
     if: "github.repository == 'numpy/numpy'"
     env:
-      PYODIDE_VERSION: 0.22.1
+      PYODIDE_VERSION: 0.23.0
```

```suggestion
      PYODIDE_VERSION: 0.23.1
```
I guess this is probably not all that is needed?  Let's just try :)